Makefile

```diff
@@ -195,7 +195,7 @@ deploy-olm: bundle-build bundle-push index-build index-push
 .PHONY: deploy-olm-testing
 deploy-olm-testing: 
 	sed -i 's/keda/keda-test/' bundle/metadata/annotations.yaml
-	sed -i 's/keda.v2.0.0/keda-test.v2.0.0/' bundle/manifests/keda.clusterserviceversion.yaml
+	sed -i 's/keda.v${BUNDLE}/keda-test.v${BUNDLE}/' bundle/manifests/keda.clusterserviceversion.yaml
```

Comment:
`$BUNDLE` is a whole image name, not the version: 
`BUNDLE = $(IMAGE_REGISTRY)/$(IMAGE_REPO)/keda-olm-operator-bundle:$(VERSION)`

$VERSION is more approprite imho